Lifeguards.cpp

```diff
@@ -0,0 +1,84 @@
+//All ten tests work
+//20 minutes
+//Difficulty: 2/10
+//It was pretty easy, I had some minor typoes but other than that is wasn't hard.
+
+
+#include <algorithm>
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+#include <queue>
+#include <stack>
+#include <map>
+#include <set>
+#include <unordered_map>
+#include <unordered_set>
+
+using namespace std;
+
+bool sortFirst(pair<int, int> p1, pair<int, int>p2) {
+	return p1.first <= p2.first;
+}
+
+int main(int argc, char* argv[]) {
+	const char* FIN = "lifeguards.in";
+	const char* FOUT = "lifeguards.out";
+	const char* inFile = argc > 1 ? argv[1] : FIN;
+	ifstream cin(inFile);
+	ofstream cout(FOUT);
+
+	int n;
+	cin >> n;
+	vector<pair<int, int>> times;
+	for (int i = 0; i < n; ++i) {
+		int x, y;
+		cin >> x >> y;
+		pair<int, int> temp;
+		temp.first = x;
+		temp.second = y;
+		times.push_back(temp);
+	}

```

Comment:
While vector works as array and also efficient, conceptually it is more complex than simple arrays. For arrays with fixed size, use simple array. The differentiation will keep code and more importantly, your understanding, clean.